apps/prairielearn/src/lib/aws.ts

```diff
@@ -148,12 +171,16 @@ export async function deleteFromS3(s3Bucket, s3Path, isDirectory = false) {
 /**
  * Get a file from S3.
  *
- * @param {string} bucket - S3 bucket name.
- * @param {string} key - The S3 target path.
- * @param {boolean} buffer - Defaults to true to return buffer.
- * @return {Promise<Buffer | import('@aws-sdk/client-s3').GetObjectOutput['Body']>} Buffer or ReadableStream type from S3 file contents.
+ * @param bucket - S3 bucket name.
+ * @param key - The S3 target path.
+ * @param buffer - Defaults to true to return buffer.
+ * @return Buffer or ReadableStream type from S3 file contents.
  */
-export async function getFromS3(bucket, key, buffer = true) {
+export async function getFromS3(
+  bucket: string,
+  key: string,
+  buffer = true,
```

Comment:
Nit: there's just one callsite of this function, and it uses the default. Consider dropping this option since it isn't used.

Reply:
I think there are two callsites in `file-store.js`.

Reply:
Ahh, right, I missed the other one. Good find.